### PR TITLE
Adjust go-live modal and notice to the latest design

### LINKED ISSUE #9916: [WC Payments NOX] Change go-live modal, notice copy
### Description

We need to change go-live modal and notice copy according to design.

Image

Image

### Designs
Design post: pfHfG4-q7-p2
Figma: xO8YdpKfaVoP2d8NvweZ5I-fi-1428_21437

### Dev notes
Reactifying the Payments settings page: pe4Hk0-1ly-p2
Populate react-powered main payments settings screen PR: https://github.com/woocommerce/woocommerce/pull/50825

### Additional context
PRD: 1wMfX8CAjDnOp24tcU-PADB0pOt3cz7mGdyZYeRBxx2Y-gdoc

### PR DESCRIPTION
**Pull request by oaratovskyi**
Fixes #9916

#### Changes proposed in this Pull Request

- Adjust go-live modal and notice to the latest design

#### Testing instructions

* Checkout to `update/9916-go-live-modal-and-notice` locally
* Open `Payments > Overview`. If you don't have an account yet, please onboard new sandbox account
* Find the go-live notice, and check that the copy aligns with the design. 
* Click it. Check that modal content is updated and it matches the design.

image

-------------------

- [x] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [x] Tested on mobile (or does not apply)

**Post merge**

- [ ] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : _Add link here / 'QA Testing Not Applicable'_
- [ ] Add or update [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) and [testing instructions for critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows), if applicable.
- [ ] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable.

### CODE REVIEW

**Comment by botwoo** [comment]
### Test the build 

#### Option 1. Jetpack Beta

- Install and activate [Jetpack Beta](https://jetpack.com/download-jetpack-beta/).
- Use this build by searching for PR number `9931` or branch name `update/9916-go-live-modal-and-notice` in your-test.site/wp-admin/admin.php?page=jetpack-beta&plugin=woocommerce-payments          

#### Option 2. Jurassic Ninja - available for logged-in A12s

:rocket: [Launch a JN site with this branch](https://jurassic.ninja/create/?jetpack-beta&shortlived&nojetpack&woocommerce&woocommerce-payments-dev-tools&branches.woocommerce-payments=update/9916-go-live-modal-and-notice) :rocket:

:information_source: Install this [Tampermonkey script](https://github.com/Automattic/woocommerce-payments/tree/develop/bin/wcpay-live-branches) to get more options.

---

Build info: 

- Latest commit: 213e3e5631c427ab5d96c9fc079f08e54a26d6aa
- Build time: 2024-12-12 10:45:49 UTC

_Note: the build is updated when a new commit is pushed to this PR._

File: changelog/update-9916-go-live-modal-and-notice [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: update
+
+Adjust go-live modal and notice to the latest design
```
Comment (anu-rock) [inline changelog/update-9916-go-live-modal-and-notice]:
```suggestion
Adjust the go-live modal to match the latest design.
```

**Review by anu-rock — APPROVED** [review]
✅ Code changes are good.
✅ The modal copy is as expected.
✅ The modal CTA "activate payments" redirects to MOX.
✅ The modal looks good in mobile view.

activate-payments-modal-mobile

Nitpick: The x close button is slightly smaller than in the design.

For reference, someone wanting to compare the new modal design/copy with the old one:
activate-payments-modal-old

**Review by dpaun1985 — APPROVED** [review]
✅ Code looks good
✅ Tested, and works as expected. 
✅ The modal looks good.

**Pull request merged**